presets/4.3/other/Tehllama_5in_2350-2750KV-Freedom_3S-4S-5S_Race_Tune.txt

```diff
@@ -0,0 +1,245 @@
+set simplified_master_multiplier = 65
+set simplified_i_gain = 85
+set simplified_d_gain = 110
+set simplified_pi_gain = 125
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 150
+set simplified_pitch_d_gain = 110
+set simplified_pitch_pi_gain = 110
+set simplified_dterm_filter = OFF
+set simplified_dterm_filter_multiplier = 140
+
+profile 2
+# profile 2 - 3S Spec
+set dterm_lpf1_dyn_min_hz = 98
+set dterm_lpf1_dyn_max_hz = 222
+set dterm_lpf1_dyn_expo = 7
+set dterm_lpf1_static_hz = 0
+set dterm_lpf2_static_hz = 210
+set vbat_sag_compensation = 99
+set iterm_relax = RPY
+set iterm_relax_cutoff = 33
+set yaw_lowpass_hz = 115
+set throttle_boost = 12
+set p_pitch = 64
+set i_pitch = 98
+set f_pitch = 220
+set p_roll = 59
+set i_roll = 89
+set d_roll = 42
+set f_roll = 201
+set p_yaw = 59
+set i_yaw = 89
+set f_yaw = 201
+set d_min_roll = 33
+set d_min_pitch = 35
+set d_max_advance = 0
+set motor_output_limit = 81
+set auto_profile_cell_count = 3
+set feedforward_jitter_factor = 12
+set feedforward_boost = 18
+set throttle_boost = 18
+set feedforward_max_rate_limit = 103
+set dyn_idle_min_rpm = 24
+set dyn_idle_p_gain = 42
+set dyn_idle_i_gain = 42
+set dyn_idle_d_gain = 42
+set simplified_master_multiplier = 105
+set simplified_i_gain = 85
+set simplified_d_gain = 105
+set simplified_pi_gain = 125
+set simplified_dmax_gain = 85
+set simplified_feedforward_gain = 160
+set simplified_pitch_d_gain = 95
+set simplified_pitch_pi_gain = 105
+set simplified_dterm_filter = OFF
+
+
+#$ OPTION BEGIN (UNCHECKED): Apply TPA Settings to all rateprofiles
+rateprofile 0
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 1
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 2
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 3
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 4
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 5
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+
+rateprofile 0
+#$ OPTION END
+
+#$ OPTION BEGIN (CHECKED): Apply TPA Settings to current rateprofile only
+set tpa_rate = 72
+set tpa_breakpoint = 1250
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED):  2350KV Motor Output Limiting (3S)
+profile 2
+set motor_output_limit = 83
+#$ OPTION END 
+
+#$ OPTION BEGIN (UNCHECKED):  2400KV Motor Output Limiting (3S)
+profile 2
+set motor_output_limit = 82
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED):  2450KV Motor Output Limiting (3S and 5S)
+profile 1
+set motor_output_limit = 98
+profile 2
+set motor_output_limit = 80
+set simplified_master_multiplier = 100
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED):  2500KV Motor Output Limiting (3S and 5S)
+profile 1
+set motor_output_limit = 96
+profile 2
+set motor_output_limit = 78
+set simplified_master_multiplier = 95
+#$ OPTION END
+
+#$ OPTION BEGIN (UNCHECKED):  2650KV Motor Output Limiting (3S and 5S)
+profile 1
+set motor_output_limit = 94
+profile 2
+set motor_output_limit = 74
+set simplified_master_multiplier = 90
```

Comment:
this won't take any effect without simplified_tuning apply following after.

Reply:
That's the part I've been struggling with - I can't even get a reliable CLI command to actually perform the simplified_tuning apply to run.

Reply:
@TehllamaFPV 
simplified_tuning apply

and in this case you don't even need to set PID values, only sliders.